app/src/main/java/pl/droidsonroids/droidsmap/feature/room/mvp/RoomUiModel.kt

```diff
@@ -3,19 +3,42 @@ package pl.droidsonroids.droidsmap.feature.room.mvp
 import pl.droidsonroids.droidsmap.base.UiModel
 import pl.droidsonroids.droidsmap.feature.room.business_logic.RoomEntity
 
+
+private const val PX_TO_METERS_FACTOR = 0.0333333333333f
+private const val LATITUDE_TO_METERS_FACTOR = 111250.33305527679f
+private const val LONGITUDE_TO_METERS_FACTOR = 70032.3813587852f
```

Comment:
Docelowo planujemy generalizację, także spojrzę chętnie na to API, ale może już nie w tym PR, i tak jest za duży